up42/auth.py

```diff
@@ -1,6 +1,7 @@
 """
 UP42 authentication mechanism and base requests functionality
 """
+
```

Comment:
Not needed

Reply:
I did not get this one, what is not needed? Are you referring to the space added?
It is automatically added by the linter in vscode.